types/testingutils/fakes.go

```diff
@@ -0,0 +1,17 @@
+package testingutils
```

Comment:
let's create a dkg file in testingutils and put all fixtures there in one place

Reply:
We can do that. I keep it here first so that we can minimize the changes to current code.

Reply:
no need to open a new file, the dkg file should have all of it in it otherwise it's gonna be hard to manage